indico/modules/events/editing/client/js/selectors.js

```diff
@@ -167,3 +172,26 @@ export const needsSubmitterConfirmation = createSelector(
     lastRevision.finalState.name === FinalRevisionState.none
 );
 export const getStaticData = state => state.staticData;
+
+export const getLastRevertableRevisionId = createSelector(
+  getDetails,
+  details => {
+    if (!details || !details.revision.length) {
+      return null;
+    }
+    const latestRevision = details.revisions.slice(-1)[0];
+    const rev = details.revisions
+      .reverse()
+      .find(r => r.finalState.name !== FinalRevisionState.none);
+
+    if (rev.id !== latestRevision.id) {
```

Comment:
Won't this fail if `rev` is null/undefined because no revision in a final state has been found?